Nodejs/Product/ProjectWizard/NodejsPackageParametersExtension.cs

```diff
@@ -19,7 +19,7 @@
 using System.Text.RegularExpressions;
```

Comment:
remove this file from changeset too - again, I think #730 should make things easier :smiley: